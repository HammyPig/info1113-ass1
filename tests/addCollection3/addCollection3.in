add book duplicates.csv 111111
LIST ALL
add collection duplicates.csv
list all
number copies