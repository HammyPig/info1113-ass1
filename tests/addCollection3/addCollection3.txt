will only add books with unadded serial numbers