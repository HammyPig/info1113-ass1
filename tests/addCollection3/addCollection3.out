user: Successfully added: To Kill a Mockingbird (Harper Lee).

user: To Kill a Mockingbird (Harper Lee)

user: 2 books successfully added.

user: To Kill a Mockingbird (Harper Lee)
To Kill a Mockingbird (Harper Lee)
The Iliad (Homer)

user: The Iliad (Homer): 1
To Kill a Mockingbird (Harper Lee): 2

user: Ending Library process.
